### DISCUSSION TITLE
Does the cookie update the component in react-cookie?

### DISCUSSION THREAD

**Discussion by victorpavlenko** [ItsBenCodes/cookies#655] [Q&A]
```
const [{accessToken}] = useCookies(["accessToken"]) 

useEffect(() => {}, [accessToken])
```

Is this possible or am I doing something wrong? 
The cookie changes but the component does not update, or is this impossible?

**Comment by arlgnv** [comment]
Hi. The component won't be updated since u have no logic inside ur effect function.
The component using useCookies hook should be provided with the up to date value for the cookie, at least in my project this is the case

**Reply by victorpavlenko** [reply]
Yes empty hook is just example. In my case it doesnt work, for reason unknown
